src/js/controller/instream-adapter.js

```diff
@@ -199,9 +199,9 @@ define([
                 playlist = [item];
             }
 
-            var _this = this;
+            _this = this;
```

Comment:
This looks like a change. Better to remove this line, or rename `var _this` to something else to fix shadowing.